### ISSUE TITLE
Any future for a Linux release?

### ISSUE THREAD

**Issue by zamlz** [cbartsch/Slippipedia#2]
Live in linux guy here.
Would love to see your app get a linux release.

**Comment by cbartsch** [comment]
Yes it's on the list. Not a Linux expert so I'll have to check out how that works some time.

**Comment by zamlz** [comment]
Awesome! I can help test it if you want, just ping me when you need me
captain!

**Comment by djanatyn** [comment]
I'm also on Linux, just wanted to add that this utility was usable for me with wine:

![neofetch](https://user-images.githubusercontent.com/523628/120359225-6e57c300-c2d5-11eb-9c90-c6ff187a1c5e.png)
![wine](https://user-images.githubusercontent.com/523628/120359227-6ef05980-c2d5-11eb-8cba-1bb73eb5f489.png)

I'd recommend giving that a try if you don't feel like waiting for a release.
I did a bit of research into QML this morning, I'm also happy to help with this issue however I can.

**Comment by derelbenkoenig** [comment]
I would also like a Linux release

**Comment by cbartsch** [comment]
Linux build included in version 2.2
